stdlib/public/Concurrency/Task.cpp

```diff
@@ -574,12 +603,16 @@ static inline bool isUnspecified(JobPriority priority) {
   return priority == JobPriority::Unspecified;
 }
 
-static inline bool taskIsUnstructured(JobFlags jobFlags) {
-  return !jobFlags.task_isAsyncLetTask() && !jobFlags.task_isGroupChildTask();
+static inline bool taskIsStructuredConcurrency(JobFlags jobFlags) {
```

Comment:
This reads weird, `taskIsStructured` would be the the usual wording and pair up well with`taskIsUnstructured`

Reply:
Good point. Will fix